docs/notebooks/Distributed_arrays_and_automatic_parallelization.md

```diff
@@ -13,19 +13,28 @@ kernelspec:
 
 +++ {"id": "PxHrg4Cjuapm"}
 
-# Parallelism with JAX
+# Distributed arrays and automatic parallelization
 
 +++ {"id": "pFtQjv4SzHRj"}
 
 **This tutorial discusses parallelism via `jax.Array`, the unified array object model available in JAX v0.4.0 and newer.**
 
-See {ref}`jax-array-migration` guide for migrating existing pre-v0.4.0 codebases to `jax.Array`.
+See [`jax-array-migration`](https://jax.readthedocs.io/en/latest/jax_array_migration.html#jax-array-migration) guide for migrating existing pre-v0.4.0 codebases to `jax.Array`.
 
-**The features required by `jax.Array` are not supported by the Colab TPU runtime at this time.**
+**The features required by `jax.Array` are not supported by the Colab TPU runtime at this time, but are available on Cloud TPU.**
 
 ```{code-cell}
-:id: 41dde63b
-
+---
+executionInfo:
```

Comment:
Remove this execution info?

Reply:
This .md file was auto-generated from the .ipynb (using jupytext), which doesn't have this execution info explicitly. Is there some way to make jupytext not generate the executionInfo? Or are you saying I should manually edit it after generating it?

Reply:
Jake told me that it doesn't work in colab bit for website it works and it works in staging rtd though.

I am fine either way though

Reply:
Sorry.. wrong place to comment.

Reply:
(Yash explained to me in chat! We can export the ipynb differently to accomplish this.)